frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -928,6 +944,60 @@ function ReaderFooter:addToMainMenu(menu_items)
                     },
                 }
             },
+            {
+                text_func = function()
+                    return T(_("Container height (%1)"), self.settings.container_height)
```

Comment:
(Dunno if these 2 `Container something` would be better in their own submenu, `Container size`?)

Reply:
None of the other stuff is in a submenu (see all the "change <stuff> formatting" at the end), so I opted for spacers between "logical" group of settings (whatever is at the beginning of the menu, spacer, dimension stuff, spacer, formatting stuff).